### DISCUSSION TITLE
RxAI: the RX.Net way to AI

### DISCUSSION THREAD

**Discussion by HowardvanRooijen** [dotnet/reactive#2176] [Show and tell]
Hi All,

I just wanted to highlight an interesting project I spotted that builds on top of Rx by @vbandi

https://github.com/vbandi/RxAI 

> RxAI is a C# library that provides a reactive programming interface for interacting with OpenAI and Azure Realtime API. It simplifies the process of creating real-time, interactive AI-powered applications using Reactive Extensions (Rx).
> ## Features:
> - Use the Realtime API from OpenAI and Azure OpenAI with a reactive (Rx.Net) programming model
> - Real-time conversation capabilities with audio input and output
> - Function calling support for extending AI capabilities
> - Built-in cost tracking and management
> - Customizable conversation options

A great example of how Rx can help with real-time scenarios.

**Comment by vbandi** [comment]
Thank you for the shoutout! LMK if you have questions / comments.
